virttest/block.py

```diff
@@ -0,0 +1,20 @@
+"""
+Utility functions to handle block related actions.
+
```

Comment:
There is a ci check error, please fix it.
if it is a one line doc string, please put it this way,
`"""Utility functions to handle block related actions."""`

Reply:
ok, thanks, will upodate~